packages/tools/src/tools/segmentation/RectangleScissorsTool.ts

```diff
@@ -91,6 +91,13 @@ class RectangleScissorsTool extends BaseTool {
    *
    */
   preMouseDownCallback = (evt: EventTypes.InteractionEventType): boolean => {
+    // if we are already drawing, means we have started with a click, and now we
```

Comment:
Couldn't a bunch of this logic be shared?  Having separate classes not sharing the logic makes it much easier to forget to update some of it.

Reply:
As discussed I'm gonna treat this PR as bug fix, and not introduce scope for now